src/common-principles.md

```diff
@@ -332,6 +347,15 @@ Derivatives can be stored/distributed in two ways:
     <dataset>/derivatives/spm-stats/sub-0001
     ```
 
+    Example of an atlas-generating pipeline, including outputs for individual subjects
+    prior to aggregation in the
+    [`MNI152NLin2009cAsym` standard space](appendices/coordinate-systems.md):
+
+    ```Plain
+    <dataset>/derivatives/atlas-generator/sub-0001
+    <dataset>/derivatives/atlas-generator/tpl-MNI152NLin2009cAsym
```

Comment:
note that following https://bids-specification.readthedocs.io/en/latest/common-principles.html#storage-of-derived-datasets this implies a "generator" variant of "atlas" pipeline... should we make it plain

```suggestion
    <dataset>/derivatives/atlasgenerator/sub-0001
    <dataset>/derivatives/atlasgenerator/tpl-MNI152NLin2009cAsym
```
?

Reply:
+1 (note that this affects most of our examples, I believe)